frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -21,32 +27,194 @@ const AlertRouting: React.FC<AlertRoutingProps> = ({ config, secret }) => {
+    count++;
+    return key === 'default' ? (
+      <span className="co-m-label__value">All (default receiver)</span>
+    ) : (
+      <React.Fragment>
+        <span className="co-m-label__key">{key}</span>
+        <span className="co-m-label__eq">=</span>
+        <span className="co-m-label__value">{value}</span>
+        {count < _.size(labels) && <React.Fragment>,&nbsp;</React.Fragment>}
+      </React.Fragment>
+    );
+  });
+  return <div className="co-m-label co-m-label--expand">{list}</div>;
+};
+
+const ReceiverTableRow: React.FC<ReceiverTableRowProps> = ({
+  obj: receiver,
+  index,
+  key,
+  style,
+  customData: routingLabels,
+}) => {
+  // filter to routing labels belonging to current Receiver
+  const receiverRoutingLabels = _.filter(routingLabels, { receiver: receiver.name });
+  return (
+    <TableRow id={index} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>{receiver.name}</TableData>
+      <TableData className={tableColumnClasses[1]}>{getIntegrationTypes(receiver)}</TableData>
+      <TableData className={tableColumnClasses[2]}>
+        {_.map(receiverRoutingLabels, (route, i) => (
+          <div key={i}>
+            <RoutingLabel labels={_.get(route, 'labels')} />
+          </div>
+        ))}
+      </TableData>
+    </TableRow>
+  );
+};
+ReceiverTableRow.displayName = 'ReceiverTableRow';
+
+const ReceiversTable: React.FC<ReceiverTableProps> = (props) => {
+  const { route, filterValue } = props;
+  const { receiver: defaultReceiver, routes } = route;
+
+  const routingLabels = getRoutinglabels(routes, {});
+  if (defaultReceiver) {
+    routingLabels.push({ receiver: defaultReceiver, labels: { default: 'all' } });
+  }
+  const EmptyMsg = () => <MsgBox title={`No Receivers match filter '${filterValue}'`} />;
+  return (
+    <Table
+      {...props}
+      aria-label="Receivers"
+      customData={routingLabels}
+      EmptyMsg={EmptyMsg}
+      Header={ReceiverTableHeader}
+      Row={ReceiverTableRow}
+      loaded={true}
+      defaultSortField="name"
+      virtualize
+    />
+  );
+};
+ReceiversTable.displayName = 'ReceiversTable';
+
+const ReceiversEmptyState: React.FC = () => (
+  <EmptyState variant={EmptyStateVariant.full}>
+    <Title size="lg">No Receivers Found</Title>
+    <EmptyStateBody>
+      Create a receiver to get OpenShift alerts through other services such as email or a chat
+      platform. The first receiver you create will become the default receiver and will
+      automatically receive all alerts from this cluster. Subsequent receivers can have specific
+      sets of alerts routed to them.
+    </EmptyStateBody>
+  </EmptyState>
+);
+
+const Receivers: React.FC<AlertManagerProps> = ({ config }) => {
+  const [receiverFilter, setReceiverFilter] = React.useState();
+  let receivers = _.get(config, ['receivers']);
+  if (receiverFilter) {
+    const fuzzyCaseInsensitive = (a, b) => fuzzy(_.toLower(a), _.toLower(b));
```

Comment:
Minor, but it would be a little more efficient to do.
```
const filterStr = _.toLower(receiverFilter);
receivers = receivers.filter((receiver) => fuzzy(filterStr, _.toLower(receiver.name)));
```

Reply:
fixed